### PR TITLE
chore: small frontend improvements

### PR DESCRIPTION
**Pull request by mds1**
- The default form data is present in dev mode but not in production, so no changes needed there.
- Add banner linking to signup page for more info https://www.scopelift.co/cove
- Add a tooltip explaining partial vs full match. The tooltip is very narrow, I think it's getting constrained by the flexbox it's in. @alexkeating @wildmolasses if either of you can fix it quickly that'd be great, but I don't think it's worth spending much time on
- Stricter lint rules

Let me know any other tweaks we want to add to this PR

### CODE REVIEW

**Comment by alexkeating** [comment]
I think the issue is the relative, and absolute, but not a 100% how to fix as I am not well versed in tailwind

**Review by alexkeating — APPROVED** [review]

File: src/lib/cove-api.ts [hunk 1]
```diff
@@ -38,12 +38,12 @@ export type SuccessfulVerification = {
   creation_bytecode: {
     object: string;
     sourceMap: string;
-    linkReferences: {};
+    linkReferences: object;
```
Comment (alexkeating) [inline src/lib/cove-api.ts:41]: Are we able to be more specific here, and the reference below?

**Review by alexkeating — COMMENTED** [review]
Looks good to me

File: src/lib/cove-api.ts (hunk 1 above)
Comment (mds1) [reply src/lib/cove-api.ts:41]: We don't use these fields so I'm not really worried about it a stronger typing here for now. For reference, this type was autogenerated by pasting output from the API into https://quicktype.io/

**Pull request merged**